diff --git a/node_modules/@types/ckeditor4/index.d.ts b/node_modules/@types/ckeditor4/index.d.ts
index cb589f8..6a9c19d 100644
--- a/node_modules/@types/ckeditor4/index.d.ts
+++ b/node_modules/@types/ckeditor4/index.d.ts
@@ -590,6 +590,9 @@ declare namespace CKEDITOR {
         autoGrow_maxHeight?: number | undefined;
         autoGrow_minHeight?: number | undefined;
         autoGrow_onStartup?: boolean | undefined;
+        autolink_commitKeystrokes?: number[] | undefined;
+        autolink_emailRegex?: RegExp | undefined;
+        autolink_urlRegex?: RegExp |undefined;
         autoUpdateElement?: boolean | undefined;
 
         baseFloatZIndex?: number | undefined;
@@ -1674,10 +1677,12 @@ declare namespace CKEDITOR {
         afterInit?(editor: editor): any;
         beforeInit?(editor: editor): any;
         init?(editor: editor): void;
+        isSupportedEnvironment?(): boolean;
         onLoad?(): any;
     }
 
     namespace plugins {
+
         namespace cloudservices {
             class cloudServicesLoader extends fileTools.fileLoader {
                 customToken: string;
@@ -1692,6 +1697,13 @@ declare namespace CKEDITOR {
             getWidgetDefinition(editor: editor, url: string): widget.definition;
         }
 
+        namespace textMatch {
+            function match(range: any, callback: any): any;
+            function getTextAndOffset(range: any): any;
+            function getAdjacentTextNodes(range: any): any;
+            function getRangeInText(range: any, start: any, end: any): any;
+        }
+
         namespace balloontoolbar {
             const PRIORITY: { [pri: string]: number };
 
